charts/graphite/templates/statefulset.yaml

```diff
@@ -20,6 +20,10 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "graphite.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+        container.audit.kubernetes.io/{{ .Chart.Name }}.allow-run-as-root: ""
```

Comment:
can you add a value for extra annotations and add this here. I don't think it is a good solution to hardcode the annotations into the template.